ui/cypress/e2e/communityTemplate.test.ts

```diff
@@ -0,0 +1,231 @@
+describe('Community Templates', () => {
```

Comment:
why not just wrap the entire thing in an `if(isFlagEnabled)` rather than setting the flag in the `config.yml`?

Reply:
@Asalem1 the tests would never run if the flag isn't turned on

Reply:
Makes sense. Is there a follow-up issue to update the `config` file for when the feature flag is removed?

Reply:
@Asalem1 yessir, it's part of the upcoming milestone.